addons/website_livechat/tests/test_chatbot_ui.py

```diff
@@ -76,10 +78,18 @@ def _check_complete_chatbot_flow_result(self):
             ("How can I help you?", operator, self.step_dispatch_operator),
             ("I want to speak with an operator", False, False),
             ("I will transfer you to a human", operator, False),
-            ("joined the channel", self.operator.partner_id, False), # human_operator has joined the channel
+            (
+                'invited <a href="#" data-oe-model="res.partner" data-oe-id="'
+                f'{operator_member.partner_id.id}">@Operator Michel</a> to the channel',
+                self.chatbot_script.operator_partner_id,
+                False,
+            ),
         ]
 
         self.assertEqual(len(conversation_messages), len(expected_messages))
+        # Operator should have the whole conversation marked as unread minus
+        # "invited operator to the channel" notification.
+        self.assertEqual(len(conversation_messages) - 1, operator_member.message_unread_counter)
```

Comment:
Would word it better as "invited" notification is not taken into account in unread counter contribution.

As the comment is written, it gives the wrong impression this notification is marked as read but not the rest, so as if marking as read is granular on each message which is not: it's a threshold from a message.